src/lib.rs

```diff
@@ -825,8 +825,9 @@ where
 {
     ///
     /// Initializes an OpenID Connect client.
-    /// If you need to configure the algorithms used for signing, ...,
-    /// do this directly on the respected components. (e.g. IdTokenVerifier)
+    /// WARNING: does not restrict id token signing algorithms,
```

Comment:
I think we can remove this comment because not calling `set_allowed_algs` below will default to `[RS256]` as indicated in the [spec](https://openid.net/specs/openid-connect-core-1_0.html#IDTokenValidation).

Reply:
Sounds good! Done.

Reply:
that happens here, reference: https://github.com/ramosbugs/openidconnect-rs/blob/a0434bb7a559e09b1de8dfbee978af7c37439eda/src/verification.rs#L148